src/app/UniquePages.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App;
+
+trait UniquePages
+{
+    /*
+    |--------------------------------------------------------------------------
+    | Unique pages for Backpack\PageManager
+    |--------------------------------------------------------------------------
+    |
+    | Each unique page has its own method, that define what fields should show up using the Backpack\CRUD API.
+    | Use snake_case for naming and PageManager will generate the page on first edit
```

Comment:
Eh? What if it's not snake_case?

Reply:
This is also a convention I just copied from default page templates behaviour.